### PR TITLE
Added welcome controller and views

### PR DESCRIPTION
**Pull request by bpresser123**
@bobbliee19

### CODE REVIEW

File: app/views/welcome/about.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<a href="http://imgur.com/wa3ydoV"><img src="http://i.imgur.com/wa3ydoV.jpg" title="source: imgur.com" /></a>
+<h1>About Blocmarks</h1>
+<p id="index-title">It's easy enough to bookmark a URL in your browser, but eventually your bookmark library may get cluttered and it may require effort to manage bookmarks. Wouldn't you also like the ability to share bookmarks with friends? That is not something you can do easily from your native browser.</p>
```
Comment (bobbilee19) [inline app/views/welcome/about.html.erb]: for easier reading i would put this message multi line so i can see all in this window...example, this below isn't example isn't exactly what you're doing but see idea...
```
@patient ||= PatientServiceIdentity.find_by!(
      service_name: PatientServiceIdentity::REFERRALS,
      patient_service_id: referral_params[:patient_uid],
    ).patient
```

File: app/views/welcome/about.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+<p id="index-title">Blocmarks will solve these problems by organizing bookmarks by topic and making them public so that other users can find them and add bookmarks to their own Blocmarks profile.</p>
```
Comment (bobbilee19) [inline app/views/welcome/about.html.erb]: same here.

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  get 'welcome/index'
+
```
Comment (bobbilee19) [inline config/routes.rb]: remove space.

File: spec/helpers/welcome_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+# describe WelcomeHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe WelcomeHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (bobbilee19) [inline spec/helpers/welcome_helper_spec.rb]: remove this file until u use it

File: spec/views/welcome/about.html.erb_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "welcome/about.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (bobbilee19) [inline spec/views/welcome/about.html.erb_spec.rb]: same here.

File: spec/views/welcome/index.html.erb_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "welcome/index.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (bobbilee19) [inline spec/views/welcome/index.html.erb_spec.rb]: ditto

**Comment by bpresser123** [comment]
@bobbilee19 updates complete.

File: app/controllers/welcome_controller.rb [hunk 7]
```diff
@@ -1,7 +1,11 @@
 class WelcomeController < ApplicationController
+
   def index
+
   end
 
   def about
+
   end
+
```
Comment (bobbilee19) [inline app/controllers/welcome_controller.rb]: no space

File: app/controllers/welcome_controller.rb [hunk 8]
```diff
@@ -1,7 +1,11 @@
 class WelcomeController < ApplicationController
+
   def index
+
   end
 
   def about
+
```
Comment (bobbilee19) [inline app/controllers/welcome_controller.rb]: no space if u have nothing in these methods

File: app/controllers/welcome_controller.rb (hunk 7 above)
Comment (bobbilee19) [inline app/controllers/welcome_controller.rb]: overall no need to worry just giving u pointers on style

File: app/views/welcome/index.html.erb [hunk 9]
```diff
@@ -1,3 +1,6 @@
-<a href="http://imgur.com/wa3ydoV"><img src="http://i.imgur.com/wa3ydoV.jpg" title="source: imgur.com" /></a>
+<a href="http://imgur.com/wa3ydoV"><img src="http://i.imgur.com/wa3ydoV.jpg" 
+title="source: imgur.com" /></a>
```
Comment (bobbilee19) [inline app/views/welcome/index.html.erb]: u dont need to do that w this one really

**Comment by bpresser123** [comment]
@bobbilee19 updates complete. Let me know when to merge or pull.

**Pull request merged**